lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -26,6 +28,7 @@ def define
         save_input_as :housing_benefit
 
         next_node do |response|
+          chosen_cap = chosen_cap
```

Comment:
I'm not sure what this line is trying to do, but I'm pretty sure it's redundant and can be removed.